assets/js/components/KeyMetrics/ChipTabGroup/index.js

```diff
@@ -0,0 +1,169 @@
+		select( CORE_FORMS ).getValue(
+			KEY_METRICS_SELECTION_FORM,
+			KEY_METRICS_SELECTED
+		)
+	);
+
+	const currentSelectionGroup = {
+		SLUG: KEY_METRICS_CURRENT_SELECTION_GROUP_SLUG,
+		LABEL: __( 'Current selection', 'google-site-kit' ),
+	};
+	const selectedCounts = {};
+	const activeMetricItems = {};
+
+	for ( const metricItemSlug in allMetricItems ) {
+		const metricGroup = allMetricItems[ metricItemSlug ].group;
+		if ( metricGroup === isActive ) {
+			activeMetricItems[ metricItemSlug ] =
+				allMetricItems[ metricItemSlug ];
+		}
+
+		if ( ! selectedCounts[ metricGroup ] ) {
+			const selectedCount = Object.keys( allMetricItems ).filter(
+				( slug ) => {
+					// Check if metric slug is in selectedMetrics, so the group
+					// count is reflected in real time as metrics are checked/unchecked.
+					if (
+						allMetricItems[ slug ].group === metricGroup &&
+						selectedMetrics.includes( slug )
+					) {
+						return true;
+					}
+
+					return false;
+				}
+			).length;
+			selectedCounts[ metricGroup ] = selectedCount;
+		}
+		// @TODO Will be updated in #9385 for current selection.
+		selectedCounts[ KEY_METRICS_CURRENT_SELECTION_GROUP_SLUG ] = 0;
+	}
+
+	const onChipChange = useCallback(
+		( slug ) => {
+			setIsActive( slug );
+		},
+		[ setIsActive ]
+	);
+
+	const chipItemRows = [
+		[ currentSelectionGroup, ...KEY_METRICS_GROUPS.slice( 0, 2 ) ],
+		[ ...KEY_METRICS_GROUPS.slice( 2 ) ],
+	];
+
+	return (
+		<div className="googlesitekit-chip-tab-group">
+			<div className="googlesitekit-chip-tab-group__tab-items">
+				{ isDesktopBreakpoint &&
+					chipItemRows.map( ( row ) => (
+						<div
+							// To avoid using indexes, key is extracted from the first grouo
+							// of each row and joined to the string "row-".
+							key={ `row-${ row[ 0 ].SLUG }` }
+							className="googlesitekit-chip-tab-group__tab-items-row"
+						>
+							{ row.map( ( group ) => (
+								<Chip
+									key={ group.SLUG }
+									slug={ group.SLUG }
+									label={ group.LABEL }
+									isActive={ group.SLUG === isActive }
+									onClick={ onChipChange }
+									selectedCount={
+										selectedCounts[ group.SLUG ]
+									}
+								/>
+							) ) }
+						</div>
+					) ) }
+				{ ! isDesktopBreakpoint &&
+					[ currentSelectionGroup, ...KEY_METRICS_GROUPS ].map(
+						( group ) => (
+							<Chip
+								key={ group.SLUG }
+								slug={ group.SLUG }
+								label={ group.LABEL }
+								isActive={ group.SLUG === isActive }
+								onClick={ onChipChange }
+								selectedCount={ selectedCounts[ group.SLUG ] }
+							/>
+						)
+					) }
+			</div>
+			<div className="googlesitekit-chip-tab-group__tab-item">
+				{ Object.keys( activeMetricItems ).map( ( slug ) => (
+					<MetricItem
+						key={ slug }
+						slug={ slug }
+						savedItemSlugs={ savedItemSlugs }
+						{ ...activeMetricItems[ slug ] }
+					/>
+				) ) }
+				{ ! Object.keys( activeMetricItems ).length && (
```

Comment:
This graphic will be actually used in full screen variation only, but since currently `current selection` is empty, I included it now, since it was quick and will visually look better until `current selection` logic is implemented